session-replacement/src/main/java/com/amadeus/session/ResetManager.java

```diff
@@ -0,0 +1,84 @@
+package com.amadeus.session;
+
+import static com.codahale.metrics.MetricRegistry.name;
+
+import com.codahale.metrics.JmxReporter;
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+
+public class ResetManager {
+  private final MetricRegistry monitoring;
+  private JmxReporter reporter; 
+  private final ErrorTracker errorTracker ;
+  
+   
+  private final Meter create_metrics ;
+  private final Meter notConnected_metrics ; 
+  private final Meter connected_metrics ;
+  
+  
+  private final SessionConfiguration configuration;
+  protected final ExecutorFacade executors;
+
+  public ResetManager(ExecutorFacade executors , SessionConfiguration configuration ) {
+    this.executors = executors;
+
+    monitoring = new MetricRegistry ();
+    
+    String SESSIONS_METRIC_MANAGER_PREFIX = "com.amadeus.session.manager";
+    String ResetManager_created_METRIC = name(SESSIONS_METRIC_MANAGER_PREFIX, "initialized");
```

Comment:
Would put it, and the others as constants (and hence all in upper case too)

Reply:
fixed